helm/dashboards/dashboards/shared/private/loki-cost-estimation.json

```diff
@@ -1032,12 +1031,12 @@
       {
         "allValue": ".*",
         "current": {
-          "selected": true,
+          "selected": false,
           "text": [
-            "All"
+            "gaia"
           ],
           "value": [
-            "$__all"
+            "gaia"
```

Comment:
Default selected cluster changed to `gaia`, sounds wrong.

Reply:
Absolutely. I'm going to revert this